benches/benches/bevy_ecs/empty_archetypes.rs

```diff
@@ -70,25 +67,26 @@ fn par_for_each(
         &A<12>,
     )>,
 ) {
-    query.par_for_each(&*task_pool, 64, |comp| {
+    query.par_iter().for_each(|comp| {
         black_box(comp);
     });
 }
 
+#[expect(
+    unused_variables,
+    reason = "`parallel` has no effect, it needs to be removed or parallel support needs to be re-added."
+)]
 fn setup(parallel: bool, setup: impl FnOnce(&mut Schedule)) -> (World, Schedule) {
```

Comment:
@james7132 or @hymm, from the commit history it looks like you both worked on this file before.

I'm not familiar with `ComputeTaskPool`, but it's no longer a resource, so I temporarily removed it. How can I re-add it to work with the current version of Bevy, or should I remove the `parallel` argument completely?

Reply:
I think it would be roughly equivalent to switch the executor kind between `SingleThreaded` and `MultiThreaded`. https://dev-docs.bevyengine.org/bevy/ecs/prelude/struct.Schedule.html#method.set_executor_kind

Reply:
Thank you for the quick response! Implemented in df26ba2a21c8b33409da8c33866066c15d933aae :)